### PR TITLE
[Ranking Criteria] add drug abuse to list of glorifying things to avoid

### PR DESCRIPTION
**Pull request by Okorin**
please dont kill me for this terminology

### Description

Adds glorifying drug abuse to the list of explicitly stated things to not upload as songs to the page, for clarity
Idk if the line ending is a problem?

### Status

- [X] Finished

### CODE REVIEW

File: wiki/Ranking_Criteria/Song_Content_Rules/en.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ _Main page: [Ranking Criteria](/wiki/Ranking_Criteria)_
 With a wide variety of music available, most tracks will be fine for use in osu!, providing that they are not:
 
 - Intensely explicit, suggestive, or otherwise extremely confrontational in regards to shocking, obscene or highly sexual content
-- Glorifying, promoting or otherwise encouraging suicide, murder, or self-harm
+- Glorifying, promoting or otherwise encouraging drug abuse, suicide, murder, or self-harm
```
Comment (TPGPL) [inline wiki/Ranking_Criteria/Song_Content_Rules/en.md]: ¿drug usage?

File: wiki/Ranking_Criteria/Song_Content_Rules/en.md [hunk 2]
```diff
@@ -35,4 +35,4 @@ Each time your submission is removed, you will be afforded an explanation by the
 You may contest this with them via PM if you so wish, or seek the opinions of others to substantiate your claim to the track's suitability. If you still cannot find common ground on this front, your claim may be escalated to a consensus vote among the current QAT, who will collectively decide if your track is acceptable or not.
 
 All users are given a **one submission leeway** before penalties are handed out due to the uncertain nature of this ruling.
-After this leeway has passed, users who continually choose to upload inappropriate tracks will face a 42 hour infringement along with the removal of their submission.
\ No newline at end of file
+After this leeway has passed, users who continually choose to upload inappropriate tracks will face a 42 hour infringement along with the removal of their submission.
```
Comment (TPGPL) [inline wiki/Ranking_Criteria/Song_Content_Rules/en.md]: 42-hour

File: wiki/Ranking_Criteria/Song_Content_Rules/en.md (hunk 1 above)
Comment (HappyStick) [reply wiki/Ranking_Criteria/Song_Content_Rules/en.md]: "drug usage" itself is too broad imo and can be construed as including maps like: https://osu.ppy.sh/beatmapsets/5823#osu/27993 

"drug abuse" in my eyes is a step further and more appropriate in this context

**Comment by Okorin** [comment]
done :D/

**Pull request merged**